src/decorator-annotator.ts

```diff
@@ -148,7 +148,8 @@ export class DecoratorClassVisitor {
    * This method looks for the place where the value that is associated to
```

Comment:
I don't get this patch's intent.  Why not just require the user to refer to the right import?